src/sql/workbench/browser/modelComponents/table.component.ts

```diff
@@ -249,7 +274,9 @@ export default class TableComponent extends ComponentBase<azdata.TableComponentP
 
 		Object.keys(this._checkboxColumns).forEach(col => this.registerPlugins(col, this._checkboxColumns[col]));
 		Object.keys(this._buttonsColumns).forEach(col => this.registerPlugins(col, this._buttonsColumns[col]));
-
+		if (this.properties['headerFilter'] === true) {
```

Comment:
Make a property accessor for this instead like other similar properties such as ariaLabel